### DISCUSSION TITLE
I deployed ory hydra on eks using helm chart

### DISCUSSION THREAD

**Discussion by naguait85** [ory/hydra#3419] [Q&A]
Hello all,
I deployed ary hydra on eks how to verify. I created public DNS names

**Comment by vinckr** [comment]
Sorry you're having trouble. We'll need more information in order to figure out what's happening here.
Can you provide some additional information, such as what were you trying to do when the error occurred?
Your setup, additional log files, the software version, your configuration and any other information that you think is helpful also helps us to solve your issue more quickly!
